packages/core/backend/sirius-components-collaborative/src/main/resources/schema/core.graphqls

```diff
@@ -26,6 +26,7 @@ type EditingContext {
   queryBasedObject(query: String!, variableName: String): Object
   queryBasedObjects(query: String!, variableName: String): [Object]
   actions: EditingContextActionsConnection!
+  object(objectId: ID!): Object
```

Comment:
As we discussed, this is needed by the frontend to retrieve the library containing an object